src/e2e/limits.rs

```diff
@@ -0,0 +1,189 @@
+use crate::contracts::{get_liquidity_by_x, get_liquidity_by_y, FeeTier, PoolKey};
+use crate::math::sqrt_price::calculate_sqrt_price;
+use crate::math::token_amount::TokenAmount;
+use crate::math::{percentage::Percentage, sqrt_price::SqrtPrice, MAX_SQRT_PRICE, MIN_SQRT_PRICE};
+use crate::token::{TokenDeployer, TokenRef};
+use crate::{InvariantDeployer, InvariantRef};
+use decimal::{Decimal, Factories};
+use odra::prelude::string::String;
+use odra::test_env;
+use odra::types::{U128, U256};
+
+fn init_dex_and_tokens_max_mint_amount() -> (InvariantRef, TokenRef, TokenRef) {
```

Comment:
If you want to use this kind of snippets.
(1) should be defined in `e2e/snippets.rs` and name it more common
(2) add `#[cfg(test)]` macro above function, similarly to: https://github.com/invariant-labs/protocol-cspr/pull/43/commits/230294d13ecaecdc9724d93c47626d6440801d77

However first decide whether this snippet is needed at all